django_cloud_deploy/config.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A module about YAML formated configuration files generation.
+
+The configuration files are expected to capture the state of Django projects. It
+should save the following attributes of a Django project:
```

Comment:
I probably wouldn't include the property names because those are controlled outside of this module so they are likely to go out of sync over time.

Reply:
Done